charts/consul/test/unit/server-service.bats

```diff
@@ -103,15 +98,6 @@ load _helpers
 #--------------------------------------------------------------------
 # annotations
 
-@test "server/Service: one annotation by default" {
-  cd `chart_dir`
-  local actual=$(helm template \
-      -s templates/server-service.yaml  \
-      . | tee /dev/stderr |
-      yq -r '.metadata.annotations | length' | tee /dev/stderr)
-  [ "${actual}" = "1" ]
-}
```

Comment:
Could this be changed to `no annotation by default` case?

Reply:
Totes!